share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -30,32 +61,32 @@ macro(rocm_create_package)
         set( CPACK_SET_DESTDIR ON CACHE BOOL "Boolean toggle to make CPack use DESTDIR mechanism when packaging" )
     endif()
 
-    if (EXISTS "/etc/os-release")
```

Comment:
We are no longer naming packages based on OS?

Reply:
For Debian package, Ubuntu 18.04 is default, ROCm support for Ubuntu16.04 is dropped.
For RPM packages el7 and el8 will be added, this is now done by the cpack.
For SLES packages lp151,etc will be added based on the SLES version. 
Please find more info 
http://confluence.amd.com/pages/viewpage.action?spaceKey=GPUCPT&title=Package+File+Naming

Reply:
When will be the packages be renamed? Post build job?

Reply:
Just because we only support Ubuntu 18.04, does not mean we can drop identifying it. Eventually, we will support other Ubuntu versions and then will need a way to distinguish packages.

Reply:
When will be the packages be renamed? Post build job?
All the package will be named in the build job. Post build job is used only for versioned packages(which is used for multiple ROCm installation)

Can you confirm when cpack will add el7 and el8? Same for SLES package, when will lp151 will be added.
Yes. The below line appends the strings el7 and el8, so that cpack can add them
string (APPEND RPM_RELEASE "%{?dist}")
This is done when the package is built on CentOS environment.

For SLES packages, lp151 will be set by the ENV variables ENV{CPACK_DEBIAN_PACKAGE_RELEASE}